### ISSUE TITLE
Sparkle update failing

### ISSUE THREAD

**Issue by tiennou** [gitx/gitx#291]
I've just got a Sparkle update popup, which downloaded something, but it fails on the install step.

<img width="732" alt="Capture d’écran 2022-02-17 à 15 02 40" src="https://user-images.githubusercontent.com/81099/154497440-18d13fdd-8fe1-4f51-97ed-ba47b54a3bcd.png">

Relevant excerpt from Console.app:
```
erreur	15:03:54.646842+0100	GitX	There is no DSA signature to check
erreur	15:03:54.646909+0100	GitX	DSA signature validation failed. The update has a public DSA key and is signed with a DSA key, but the public key doesn't match the signature. The update will be rejected.
par défaut	15:03:54.683646+0100	GitX	Warning: Expected min height of view: (<NSButton: 0x7f93e981e540>) to be less than or equal to 30 but got a height of 32.000000. This error will be logged once per view in violation.
erreur	15:03:55.958467+0100	GitX	Error: An error occurred while extracting the archive. Please try again later. The update is improperly signed. (URL (null))
```

**Comment by tiennou** [comment]
Good job on bringing the update system back online though 🍾. I suspect the version I have might be using Rowan's key for verification, or even none at all, since I've been the only person to have working dev builds for a loooong while, thanks to the build system, but you've — painfully — been made aware of that.

**Comment by hannesa2** [comment]
Welcome back ! I see you are at least reading here (sometimes) what happens. 
You are the guy with the most knowledge here.

**Comment by hannesa2** [comment]
Btw, I see the icons are not beautiful, I think we should remove them. The most ugly are the features

**Comment by hannesa2** [comment]
This repository is good demonstration how important several admins are, otherwise it's dedicated for being a death repository.

@tiennou Please have a look to https://github.com/libgit2/objective-git/issues/717 and please think about to share your power/permissions

**Comment by claytonrcarter** [comment]
Yes, 😞 🙈 it's a work in progress, I guess. We have the build process updated and reliably working, and now we have the Sparkle appcast also updating regularly, but we have not figured out the DSA/edDSA signature part. Even if we did, Sparkle would be installing an unsigned binary (another TBD item, and a thorny one, I suspect) which might cause other issues.

Thanks for logs, too. I hadn't looked there. @hannesa2 it looks like *this* issue is more about DMG signing than about app signing.

@tiennou Thanks again for all of your work on this app over the years.
